services/brig/test/integration/API.hs

```diff
@@ -598,9 +642,14 @@ testPasswordChange brig = do
 testSendActivationCode :: Brig -> Http ()
 testSendActivationCode brig = do
     p <- randomPhone
+    -- Code for pre-verification
     send $ RequestBodyLBS . encode $ object ["phone" .= fromPhone p]
+    e <- randomEmail
+    -- Code for pre-verification
+    send $ RequestBodyLBS . encode $ object ["email" .= fromEmail e]
```

Comment:
instead of `send $ RequestBodyLBS . encode $ object ["email" .= fromEmail e]`, use the function introduced in this PR `requestActivationCode` from Util?

Reply:
Yes, this was an oversight. Thanks!